start-configuration

```diff
@@ -22,8 +15,16 @@ if [ ! -e /data/eula.txt ]; then
     echo ""
     exit 1
   fi
+
+  echo "# Generated via Docker on $(date)" > eula.txt
+  echo "eula=$EULA" >> eula.txt
```

Comment:
...and this to
```suggestion
  echo "eula=${EULA^^}" >> eula.txt
```

Reply:
Thanks for the hint! I also thought about allowing mixed case (or even `yes`) and just hard coding the `TRUE` like so:

    echo "eula=TRUE" >> eula.txt

But I didn't know if this was legally allowed or if there is any requirement, that the user has to type *exactly* the text that has to be written in the eula file. Therefore I decided to not change this and just use the value the user typed, but disallow anything that would not work.

If you don't think there is any problem in doing it the way you proposed, I'm totally fine with it and I would also prefer your solution. 😉

Reply:
I think just sticking to variations of “true” is sufficient flexibility. The combination of the extra check you have added and the uppercase’ing will guard and simplify giving Minecraft the right value.

Reply:
I've added another commit in which the value is transformed to upper case once before it is used so it does not have to be done every time the value is used.